openassessment/assessment/models/staff.py

```diff
@@ -0,0 +1,117 @@
+"""
+Models for managing staff assessments.
+"""
+from datetime import timedelta
+
+from django.db import models, DatabaseError
+from django.utils.timezone import now
+
+from openassessment.assessment.models.base import Assessment
+from openassessment.assessment.errors import StaffAssessmentInternalError
+
+
+class StaffWorkflow(models.Model):
+    """Internal Model for tracking Staff Assessment Workflow
+
+    This model can be used to determine the following information required
+    throughout the Staff Assessment Workflow:
+
+    1) Get next submission that requires assessment.
+    2) Does a submission have a staff assessments?
+    4) Does this staff member already have a submission open for assessment?
+    5) Close open assessments when completed.
+
+    """
+    # Amount of time before a lease on a submission expires
+    TIME_LIMIT = timedelta(hours=8)
```

Comment:
Currently it's not configurable for `PeerWorkflow`s, so I kept the logic mostly the same.